platform-sdk/swirlds-virtualmap/src/main/java/module-info.java

```diff
@@ -13,7 +13,11 @@
     exports com.swirlds.virtualmap.internal to
             com.swirlds.virtualmap.test.fixtures;
     exports com.swirlds.virtualmap.internal.cache to
```

Comment:
internal api should stay internal

Reply:
Please see: https://github.com/hashgraph/hedera-services/pull/15562/commits/c8d72fde119e95291ec5f0a87ebb7493275fa92a